### ISSUE TITLE
Fresh project fails with kapt error "tools.jar is absent"

### ISSUE THREAD

**Issue by breucode** [jhipster/jhipster-kotlin#320]
## Overview of the issue

The build of a fresh project fails with the following error message:
```
> Task :kaptGenerateStubsKotlin FAILED
e: [kapt] 'com.sun.tools.javac.util.Context' class can't be found ('tools.jar' is absent in the plugin classpath). Kapt won't work.
```

## Reproduce the error

Create a fresh project with the current generator-jhipster-kotlin version.

After the project was created, run `./gradlew`.

## KHipster Version(s)

1.14.0

## Fix

This problem is fixed by upgrading kotlin to 1.6.10 (`kotlin_version=1.6.10` in `gradle.properties`).

-   [ X ] Checking this box is mandatory (this is just to show you read everything)

**Comment by sendilkumarn** [comment]
Interesting! The CI works fine in this case. 

Can you post `khipster info` command?

**Comment by sendilkumarn** [comment]
This should be closed via #333 

Can you try with the generator from the main branch to check? 

I am closing this for now but please feel free to reopen if needed.
